interface/modules/custom_modules/oe-module-claimrev-connect/ModuleManagerListener.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * Class to be called from Laminas Module Manager for reporting management actions.
+ * Example is if the module is enabled, disabled or unregistered ect.
+ *
+ * The class is in the Laminas "Installer\Controller" namespace.
+ * Currently, register isn't supported of which support should be a part of install.
+ * If an error needs to be reported to user, return description of error.
+ * However, whatever action trapped here has already occurred in Manager.
+ * Catch any exceptions because chances are they will be overlooked in Laminas module.
+ * Report them in the return value.
+ *
+ * @package   OpenEMR Modules
+ * @link      https://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2024 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+/*
+ * Do not declare a namespace
+ * If you want Lamina's manager to set namespace set it in getModuleNamespace
+ * otherwise uncomment below and set path.
+ *
+ * */
+
+/*
+    $classLoader = new \OpenEMR\Core\ModulesClassLoader($GLOBALS['fileroot']);
+    $classLoader->registerNamespaceIfNotExists("OpenEMR\\Modules\\ClaimRevConnector\\", __DIR__ . DIRECTORY_SEPARATOR . 'src');
+*/
+
+use OpenEMR\Core\AbstractModuleActionListener;
+use OpenEMR\Modules\WenoModule\Services\ModuleService;
+
+/* Allows maintenance of background tasks depending on Module Manager action. */
+
+class ModuleManagerListener extends AbstractModuleActionListener
```

Comment:
@claimrevolution 
I added this to your claim module which I setup to handle background service tasks.
It tested well for me!
Recommend you review and test!